src/schema/v2/artwork/collectorSignals.ts

```diff
@@ -107,6 +107,25 @@ export const CollectorSignals: GraphQLFieldConfig<any, ResolverContext> = {
         description: "Bid count on lots open for bidding",
         deprecationReason: "Use nested field in `auction` instead",
       },
+      curatorsPick: {
+        type: GraphQLBoolean,
+        description:
+          "Artwork is part of either the Curators' Pick Emerging or Blue Chip collections",
+        resolve: async (artwork, {}, ctx) => {
+          const CURATED_COLLECTION_SLUGS = [
+            "curators-picks-blue-chip-artists",
+            "curators-picks-emerging-artists",
+          ]
+
+          const checks = await Promise.all(
```

Comment:
Question: I think if one of the requests fails, `Promise.all` would be rejected and I guess the field would return `null`. In cases like intermittent issues it might be fine, but in cases like the hard-coded slugs above get renamed/retired upstream, this signal could be silently broken. I wonder if there is tool to help monitor and alert when it's not doing its job.

Reply:
It wouldn't necessarily be silently broken - the `errors` would be included in the Metaphysics response, and they may even be logged to Sentry (unless the error was a 401/403/404 from upstream - those get skipped in Sentry reporting).

I would actually say the other way around - if the field was nullable, and then you had a `try/catch { return false}` - _that's_ a pattern where errors get silenced and it's possible for the thing to just be broken w/o anyone knowing.

Reply:
Should we re-enable Sentry for Metaphysics? 🙈 I think at least we should track things on the product level, e.g. tracking impression for each signal on the data side, and maybe, if possible, notify for anomaly.